README.md

```diff
@@ -113,6 +113,196 @@ The below Environment variables are optional. If not provided a manual Applicati
 |`BOLD_SERVICES_USER_EMAIL`|Yes|It should be a valid email.|
 |`BOLD_SERVICES_USER_PASSWORD`|Yes|It should meet our password requirements.|
 
+## Environment variables for configuring `Branding` in backend
+The below Environment variables are optional. If not provided Bold BI will take the default configured values.
+
+<table>
+   <tr>
+      <td>
+       <b>Name</b>
+      </td>
+      <td>
+       <b>Description</b>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       BOLD_SERVICES_BRANDING_MAIN_LOGO
+      </td>
+      <td>   
+       This is header logo for the applicationand the preferred image size is 40 x 40 pixels.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       BOLD_SERVICES_BRANDING_LOGIN_LOGO
+      </td>
+      <td>     
+       This is login logo for the application and the preferred image size is 200 x 40 pixels.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       BOLD_SERVICES_BRANDING_EMAIL_LOGO
+      </td>
+      <td>     
+       This is email logo, and the preferred image size is 200 x 40 pixels.
```

Comment:
This is an email logo, and the preferred image size is 200 x 40 pixels.

Reply:
concern addressed